docs/proposals/volume-provisioning.md

```diff
@@ -59,18 +59,15 @@ types of volumes within a single cloud.
 
 One of our goals is to enable administrators to create out-of-tree
 provisioners, that is, provisioners whose code does not live in the Kubernetes
-project.  Our experience since the 1.2 release with dynamic provisioning has
-shown that it is impossible to anticipate every aspect and manner of
-provisioning that administrators will want to perform.  The proposed design
-should not prevent future work to allow out-of-tree provisioners.
+project.
 
 ## Design
 
 This design represents the minimally viable changes required to provision based on storage classe configuration.  Additional incremental features may be added as a separte effort.
 
 We propose that:
 
-1.  For the base impelementation storage class and volume selectors are mutually exclusive.
+1.  For the base implementation storage class and volume selectors are mutually exclusive for now.
 
 2.  An api object will be incubated in extensions/v1beta1 named `storage` to hold the a `StorageClass`
     API resource. Each StorageClass object contains parameters required by the provisioner to provision volumes of that class.  These parameters are opaque to the user.
```

Comment:
opaque to the user or to the system?  The user (admin?) would know what 'gp2' is but the system wouldn't.